### ISSUE TITLE
question regarding alpha being used in generating trimap

### ISSUE THREAD

**Issue by ghazni123** [huochaitiantang/pytorch-deep-image-matting#40]
Hi

I have question regarding alpha needed during testing phase. Per my understanding, paper states that for testing/inference only original image & corresponding trimap are needed and encoder/decoder framework will predict the alpha. On the other hand Ground truth alpha (generated by photoshop) is required for training. 

I am aware that originally authors calculated alpha manually for all 431 objects and after composing distributed among training and testing sets. So training and testing both gets high quality alpha produced manually. This is fine is testing images of authors are used.

However if I chose another image from internet and want to pass through network (per paper I should only provide image and correponding trimap)

Why alpha being passed during testing phase to generate trimap? and how do I get that alpha? Is it the case that mask generated by any segmentation framework can be used as initial/rough alpha as input to generate_trimap and by running through the network will produce/predict better alpha?

Many thanks.
